addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -425,7 +425,7 @@ def _import_fill_invoice_allowance_charge(self, tree, invoice_form, journal, qty
                     invoice_line_form.price_unit = float(amount_node.text) * charge_factor * qty_factor
 
                 invoice_line_form.tax_ids.clear()  # clear the default taxes applied to the line
-                tax_xpath = './{*}TaxCategory/{*}Percent' if is_ubl else './{*}CategoryTradeTax/{*}RateApplicablePercent'
+                tax_xpath = '../{*}Item/{*}ClassifiedTaxCategory/{*}Percent' if is_ubl else './{*}CategoryTradeTax/{*}RateApplicablePercent'
```

Comment:
same here, Item's are only present on the lines, it doesn't make sense on the document level